packages/smarthr-ui/src/components/Heading/stories/Heading.stories.tsx

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable smarthr/a11y-heading-in-sectioning-content */
+import React from 'react'
+
+import { Stack } from '../../Layout'
+import { Heading } from '../Heading'
+
+import type { Meta, StoryObj } from '@storybook/react'
+
+export default {
+  title: 'Text（テキスト）/Heading',
+  component: Heading,
```

Comment:
賛否ありそうですが、subcomponents として PageHeading を書いてもよさそう。

```suggestion
  component: Heading,
  subcomponents: { PageHeading },
```